android/src/main/java/com/reactnativehmssdk/HMSManager.kt

```diff
@@ -487,6 +697,10 @@ class HMSManager(reactContext: ReactApplicationContext) :
   @RequiresApi(Build.VERSION_CODES.O)
   private fun readableMapToPipParamConfig(data: ReadableMap?): PipParamConfig {
     var aspectRatio: Pair<Int, Int>? = null;
```

Comment:
set `aspectRatio` default `[16, 9]` here

Reply:
done in commit - 7b46fb1667547a90382fa37dd0d45281ff5f0203